helm/values.yaml

```diff
@@ -0,0 +1,409 @@
+  revokeEnd: https://auth.example.com/oauth2/revoke
+
+config:
+  http: 
+    # MAS URL
+    base: https://auth.example.com/
+    # OIDC issuer advertised by the service. Defaults to `public_base`
+    issuer: https://auth.example.com
+  listeners:
+    resources:
+      - name: discovery # Serves the .well-known/openid-configuration document    
+      - name: human # Serves the human-facing pages, such as the login page
+      - name: oauth # Serves the OAuth 2.0/OIDC endpoints
+      - name: compat # Serves the Matrix C-S API compatibility endpoints
+      - name: graphql # Serve the GraphQL API used by the frontend, and optionally the GraphQL playground
+        playground: false
+      - name: assets # Serve the given folder on the /assets/ path
+        path: /usr/local/share/mas-cli/assets/
+    binds:
+      # use either address, host/port (hp) or socket
+      type:
+        address: 
+          enabled: false
+        hp:
+          enabled: true
+        socket: 
+          enabled: false 
+      address: '[::]:8080'
+      host:
+        name: "0.0.0.0"
+        port: '8080'
+      #socket: /tmp/mas.sock
+    # currently not configured  
+    proxy_protocol: false
+  trusted_proxies:
+  - 10.0.0.0/10
+  - 127.0.0.1/8
+  - fd00::/8
+  - ::1/128
+
+# must be PostgreSQL
+db:
+  host: postgres
+  port: 5432
+  user: user1
+  pass: password
+  dbName: database
+  min_connections: 0
+  max_connections: 10
+  connect_timeout: 30
+  idle_timeout: 600
+  max_lifetime: 1800
+
+# Settings related to the connection to the Matrix homeserver
+matrix:
+  # `server_name` in the Synapse configuration file
+  hs: example.com
+  secret: someRandomSecret
+  hs_url: https://synapse.example.com
+
+# Allows loading custom templates
+# From where to load the templates
+# This is relative to the current working directory, *not* the config file
+templates:
+  enabled: false
+  path: /usr/local/share/mas-cli/templates/
+  # Path to the frontend assets manifest file
+  assets: /to/manifest.json
+
+clients:
+  # Confidential client
+  # Each client_id must be a ULID https://github.com/ulid/spec
+  sec_id: 0000000000000000000SYNAPSE
+  secAuthMethod: client_secret_post
+  client_secret: secret
+  redirect_uris:
+    - http://localhost:1234/callback
+  # Public client
+  pub_id: 0000000000000000000SYNAPSX
+  pubAuthMethod: none
+
+secrets:
+  # Encryption secret (used for encrypting cookies and database fields)
+  # This must be a 32-byte long hex-encoded key
+  encrypt: 
+  # It needs at least an RSA key to work properly
+  # Following keys are supported
+  # RSA
+  # ECDSA with the P-256 (prime256v1) curve
+  # ECDSA with the P-384 (secp384r1) curve
+  # ECDSA with the K-256 (secp256k1) curve
+
+  # you should use the mas-cli with the command `mas-cli config generate` to get a new set of secrets
+  keys:
+    - kid: keyid#1
+      key: |-
+
+    - kid: keyid#2
+      key: |-
+
+    - kid: keyid#3
+      key: |
+
+    - kid: keyid#4
+      key: |
```

Comment:
Same here: you could generate a RSA key and a ECDSA P-256 key with Helm and that would be enough